src/library-authoring/component-info/ComponentInfo.tsx

```diff
@@ -189,7 +192,10 @@ const ComponentInfo = () => {
             >
               {intl.formatMessage(messages.publishComponentButtonTitle)}
             </Button>
-            <ComponentMenu usageKey={usageKey} />
+            <ComponentMenu
+              usageKey={usageKey}
+              unitsData={(componentHit as ContentHit[])?.[0]?.units}
+            />
```

Comment:
@ChrisChV It would be simpler to call `useComponentsFromSearchIndex` inside of `ComponentMenu` or `ComponentDeleter` (preferably former) component instead of passing through props.

Reply:
Refactored here: https://github.com/openedx/frontend-app-authoring/pull/1906/commits/6291f67dbe69d4776127c138b0b1d9f22bb9c170